web/frontend/src/components/slam/index.svelte

```diff
@@ -97,6 +99,14 @@
           ]);
           nextPose = response.pose;
         }
+
+        if (adjustPointCloudWithPosition) {
+            const command = "slam_path_point_cloud"
+            let slamPathStruct = await slamClient.doCommand({"slam_path_point_cloud": ""})
+            if (slamPathStruct[command]) {
+              slamPath = new Uint8Array(slamPathStruct[command] as Uint8Array)
```

Comment:
What type of data is returned from the DoCommand? This is making lots of assumptions and it would be great to try to safely parse this.